README.md

```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
 
 
 ## Contents
-- [Description](#description)
-- [Prerequisites](#prerequisites)
-- [Instructions](#instructions)
-  - [Getting Started](#getting-started)
-  - [Cleanup](#cleanup)
-- [Configuration](#configuration)
-- [Known Issues](#known-issues)
-- [Get Support](#get-support)
-- [Contributing](#contributing)
-- [Upcoming Changes](#upcoming-changes)
-- [License](#license)
+- [i40-aas](#i40-aas)
+  - [Contents](#Contents)
+  - [Main Features](#Main-Features)
+  - [Context - Industry 4.0](#Context---Industry-40)
+  - [Prerequisites](#Prerequisites)
+  - [Instructions](#Instructions)
+    - [Getting Started](#Getting-Started)
+    - [Cleanup](#Cleanup)
+  - [Configuration](#Configuration)
+  - [Known Issues](#Known-Issues)
+  - [Get Support](#Get-Support)
+  - [Contributing](#Contributing)
+  - [Upcoming Changes](#Upcoming-Changes)
+  - [License](#License)
 
 
+## Main Features
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
+
+- Standardized and secure communication interfaces and API to access Asset Administration Shells of the digital representation of a physical asset (Digital Twin), that enable:
+  - use of a single AAS information model
+  - secure access to asset information from cloud and edge
```

Comment:
we do not have security full implemented yet. We should only list the current status here.

Reply:
updated